api/src/main/java/io/minio/MinioClient.java

```diff
@@ -17,40 +17,19 @@
 
 package io.minio;
 
+import com.fasterxml.jackson.databind.type.MapType;
```

Comment:
You could have a separate subproject called `MinioAdmin` instead of adding this to `MinioClient` which is meant to do S3 APIs.